Makefile

```diff
@@ -108,10 +108,18 @@ checks: flake8 black-check mypy isort-check docs
 
 lint: isort black
 
-tests:
+tests: unit-tests integration-tests
+
+unit-tests:
 	@echo "🥫 Running tests …"
 	# run tests in worker to have more memory
-	${DOCKER_COMPOSE} run --rm workers poetry run pytest tests
+	${DOCKER_COMPOSE} run --rm workers poetry run pytest --cov-report xml --cov=robotoff tests/unit 
+
+integration-tests: dev
+	@echo "🥫 Running integration tests …"
+	# run tests in worker to have more memory
+	${DOCKER_COMPOSE} run --rm workers poetry run pytest -vv --cov-report xml --cov=robotoff --cov-append tests/integration
+	${DOCKER_COMPOSE} down
```

Comment:
The way we run integrations tests, without changing database name, means we are destructive on the dev desktop. 

I think we can tackle that as a separate issue, but do you agree this is a problem ? (I do find it an important problem)

Reply:
I do :-)

Reply:
Alright, let's leave that as topic for a new PR?

Reply:
ok.

Reply:
I opened an issue : https://github.com/openfoodfacts/robotoff/issues/490